Source/CarthageKit/Project.swift

```diff
@@ -29,7 +49,9 @@ private let CarthageUserCachesURL: NSURL = {
 		NSLog("Warning: No Caches directory could be found or created: \(error.value.localizedDescription). (\(error.value))")
 	}
 
-	return URL.value?.URLByAppendingPathComponent(CarthageKitBundleIdentifier, isDirectory: true) ?? fallbackDependenciesURL
+	let dependenciesURL = URL.value?.URLByAppendingPathComponent(CarthageKitBundleIdentifier, isDirectory: true)
+	return dependenciesURL ?? fallbackDependenciesURL
```

Comment:
While you're at it, would you mind moving these `return`s into the `switch` above? :pray: